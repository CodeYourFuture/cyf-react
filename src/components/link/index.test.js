import React from 'react';
import { shallow } from 'enzyme';

import CYFLink from './';

describe('CYFLink Component', () => {
  it('renders internal links correctly', () => {
    expect(
      shallow(<CYFLink path="/apply" title="Become a Mentor" />),
    ).toMatchSnapshot();
  });

  it('renders external links correctly', () => {
    expect(
      shallow(<CYFLink path="https://codeyourfuture.io" title="Homepage" />),
    ).toMatchSnapshot();
  });
});
