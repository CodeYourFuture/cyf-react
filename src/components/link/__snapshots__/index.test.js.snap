// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CYFLink Component renders external links correctly 1`] = `
<div
  className="section-bottom-link"
>
  <a
    className="big-link-3 btn"
    href="https://codeyourfuture.io"
    rel="noopener noreferrer"
    target="_blank"
  >
    Homepage
  </a>
</div>
`;

exports[`CYFLink Component renders internal links correctly 1`] = `
<div
  className="section-bottom-link"
>
  <Link
    className="big-link-3 btn"
    replace={false}
    to="/apply"
  >
    Become a Mentor
  </Link>
</div>
`;
